doozer/doozerlib/cli/fbc.py

```
@@ -329,7 +329,19 @@ async def run(self):
+        # Mint a per-invocation GitHub App token for git-clone auth
+        git_auth_secret = await merger._konflux_client.ensure_git_auth_secret(
+            namespace=self.konflux_namespace,
+        )
+        await merger.run(fragments, target_index, git_auth_secret=git_auth_secret)
+
+        # Best-effort cleanup of transient git-auth secrets older than 1 day
+        try:
+            await merger._konflux_client.cleanup_stale_git_auth_secrets(
+                namespace=self.konflux_namespace,
+            )
+        except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)